### DISCUSSION TITLE
How to use controlnet mask?

### DISCUSSION THREAD

**Discussion by pflky** [lllyasviel/stable-diffusion-webui-forge#2338] [General]
I swear I figured this out before, but my issue is that if I use the "use mask" option with controlnet, it ignores controlnet and even the mask entirely. Same for if I inpaint the mask directly on the image itself in controlnet. It just generates as if you're using a txt2img prompt by itself. It seems like nothing works.

**Comment by AnabelleKM** [comment]
I think that might be because of the same problem why Batch Upload only looks at the first image and not the entire batch.
ControlNet is in a fragile state at the moment.
